scripts/start-services-no-ui.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -euo pipefail
+
+readonly SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+readonly UI_NAME=make-recall-decision-ui
+readonly API_NAME=make-recall-decision-api
+readonly UI_DIR="${SCRIPT_DIR}/../../${UI_NAME}"
+readonly API_DIR="${SCRIPT_DIR}/../../${API_NAME}"
+
+pushd "${API_DIR}"
+printf "\n\nBuilding/starting API components...\n\n"
+docker-compose pull
+docker-compose build
+docker-compose up -d
+popd
+
+pushd "${UI_DIR}"
+printf "\n\nBuilding/starting UI components...\n\n"
+docker-compose pull
+docker-compose build
+docker-compose up --scale make-recall-decision-ui=0
+popd
+
+function wait_for {
+  printf "\n\nWaiting for %s to be ready.\n\n" "${2}"
+  until curl -s --fail "${1}"; do
+    printf "."
+    sleep 2
+  done
+}
+
+wait_for "http://localhost:9090/auth/health/ping" "hmpps-auth"
+wait_for "http://localhost:3000/ping" "${UI_NAME}"
```

Comment:
Thia will fail unless you start up UI in a separate terminal session - might be best to remove this check and add something to the closing output to say "and now start up UI by doing ..."

Reply:
removed